packages/block-editor/src/components/height-control/index.js

```diff
@@ -164,6 +164,7 @@ export default function HeightControl( {
 				<FlexItem isBlock>
 					<Spacer marginX={ 2 } marginBottom={ 0 }>
 						<RangeControl
+							__next40pxDefaultSize
```

Comment:
Can be seen in the Group block ▸ Styles ▸ Dimensions ▸ Minimum Height. No visual change because this is a standalone slider-only RangeControl instance.

<img src="https://github.com/user-attachments/assets/db0d669b-7317-4dd4-86c8-d543a94ea525" alt="HeightControl, after" width="259">